app/js/components/tabs/tabcomponents/patientComponent.jsx

```diff
@@ -180,10 +207,10 @@ class PatientComponent extends Component {
                         {
                             this.state.toDisplay.map(patient => {
                                 return (
-                                    <tr key={patient.uuid}>
-                                        <td>{patient.person.display}</td>
-                                        <td>{patient.person.age}</td>
-                                        <td>{patient.person.gender}</td>
+                                    <tr key={shortId.generate()}>
+                                        <td>{`${patient.Firstname} ${patient.Lastname}`}</td>
```

Comment:
That was actually my fault. There's a statement in the jsonHelper class [here](https://github.com/openmrs/openmrs-owa-cohortbuilder/blob/master/app/js/helpers/jsonHelper.js#L66). The way the adhocquery resource was designed you specify the name of the columns you want and it adds it to the property. I'll fix that right away